package.json

```diff
@@ -104,9 +104,9 @@
     "check": "eslint --ignore-path .gitignore --ignore-pattern node_modules --quiet .",
     "fix": "eslint --ignore-path .gitignore --ignore-pattern node_modules --quiet . --fix",
     "postinstall": "make post-install",
-    "test": "jest --forceExit",
-    "test:watch": "jest --watch",
-    "test:coverage": "jest --coverage"
+    "test": "TZ=utc jest --forceExit",
```

Comment:
Ideally, we could set this at runtime and test variants of timezones, but I couldn't find a reliable way to do that in a bug fix timeframe.